src/aap_eda/services/activation/tee_system_logger.py

```diff
@@ -0,0 +1,76 @@
+#  Copyright 2025 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import logging
+import sys
+from datetime import datetime, timezone
+
+from aap_eda.services.activation.db_log_handler import DBLogger
+
+EXCEPTIONS_TO_CATCH = (
+    OverflowError,
+    OSError,
+    ValueError,
+    TypeError,
+    AttributeError,
+    KeyError,
+)
+
+LOGGER = logging.getLogger(__name__)
+
+
+class TeeSystemLogger(DBLogger):
+    """
+    A logging class that extends DBLogger.
+
+    It adds log messages both to the database and the system logger.
+    """
+
+    @staticmethod
+    def _convert_to_asctime(timestamp: int) -> str:
+        """Convert a UNIX timestamp to asctime format in UTC."""
+        return datetime.fromtimestamp(timestamp, tz=timezone.utc).strftime(
+            "%Y-%m-%d %H:%M:%S,%f"
+        )[:-3]
+
+    def flush(self):
+        try:
+            for buffer in self.activation_instance_log_buffer:
+                line = buffer.log
+                rulebook_timestamp = self._convert_to_asctime(
+                    buffer.log_timestamp
+                )
+                # Determine the log level
+                if "ERROR" in line:
```

Comment:
The criteria to determine the log level might be tricky and not reliable because for example, other level could contain "ERROR" string and it would be considered of that level wrongly. 
I suggest a better parsing or at least limit the search for the 40 first characters. WDYT?